test/cluster/cluster-test.sh

```
@@ -27,16 +27,21 @@ function cluster_setup() {
   pwd
   # see DOCKER_IMAGES_TXT in pipeline-steps.go for env var
   # replace all docker image tags with previously built candidate images
+  pushd base
+  set +o pipefail
```

Why do we need to disable this and +e?

---

In the list we are iterating over there are some images that are not in the k8s cluster (`ubuntu-ignite` & `server`). We shouldn't fail the whole pipeline when we hit these

---

Couldnt we instead grep those out of the list? or `if this then continue`. I believe this opens up for other things not being deployed and failing silently.

---

True, we could add a step here to either exclude containers not in the K8s deployment or specifically skip those containers. 
The risk here is if we fail to swap out an image it will use the one in head.